pandas/io/formats/style_render.py

```diff
@@ -209,37 +211,38 @@ def _translate_header(
 
         head = []
         # 1) column headers
-        for r in range(self.data.columns.nlevels):
-            index_blanks = [
-                _element("th", blank_class, blank_value, not self.hidden_index)
-            ] * (self.data.index.nlevels - 1)
-
-            name = self.data.columns.names[r]
-            column_name = [
-                _element(
-                    "th",
-                    f"{blank_class if name is None else index_name_class} level{r}",
-                    name if name is not None else blank_value,
-                    not self.hidden_index,
-                )
-            ]
-
-            if clabels:
-                column_headers = [
+        if not self.hidden_colheads:
```

Comment:
all that was changed here was to add `if not self.hidden_colheads:`, the other additions and removals in comparison window are misleading.